splitter/use-100-percent-viewport/use-100-percent-viewport/Shared/MainLayout.razor

```diff
@@ -0,0 +1,61 @@
+﻿@layout TelerikLayout
+
+@inherits LayoutComponentBase
+
+<style>
+    /* CSS rules to prepare the whole 100% height layout
+    Note that for 100% height to work, all elements up the DOM tree must have a defined height in percent or pixels
+    */
+    html, body {
+        height: 100%;
+        margin: 0;
+        overflow: hidden;
+    }
+</style>
+
+<TelerikSplitter Height="100%" Width="100%" Orientation="@SplitterOrientation.Vertical">
+    <SplitterPanes>
+
+        @*Header*@
+        <SplitterPane Resizable="false" Size="100px">
+            <div style="height:100%; background: green;">
+                @*Sample styles to showcase where content renders*@
+                <h1>I am the header</h1>
+            </div>
+        </SplitterPane>
+
+        @*Main content*@
+        <SplitterPane>
+            <TelerikSplitter Width="100%" Height="100%" Class="k-pane-flex">
+                <SplitterPanes>
+
+                    @*Left navigation*@
+                    <SplitterPane Size="300px" Collapsible="true">
+                        <div style="height:100%; background: yellow;">
+                            SIDEBAR
+                            <NavMenu></NavMenu>
+                        </div>
+                    </SplitterPane>
+
+                    @*Page (main) content*@
+                    <SplitterPane>
+                        @*Automatic scrollbars for the content, you can choose where you want scolling to happen*@
+                        <div style="height: 100%; overflow: auto;">
+                            @Body
+                        </div>
+                    </SplitterPane>
+
+                </SplitterPanes>
+            </TelerikSplitter>
+        </SplitterPane>
+
+        @*Footer*@
+        <SplitterPane Resizable="false" Size="100px">
+            <div style="height:100%; background: red;">

```

Comment:
```suggestion
            <div style="height:100%; background: yellow;">

```

Reply:
I want to keep the header, footer and sidebar distinct in terms of colors, so I will keep this red, the sidebar is yellow.

Reply:
Indeed, I missed the yellow color. How about something less flamboyant, e.g. 
```suggestion
            <div style="height:100%; background: #f99;">

```

Reply:
I trust your judgement on colors far more than mine :)